contributing.md

```diff
@@ -0,0 +1,65 @@
+
+# Contribution Workflow
+
+The following is a summary of the ideal contribution flow. Please, note that Pull Requests can also be rejected by the maintainers when appropriate.
+
+    ┌───────────────────────┐
+    │                       │
+    │    Open an issue      │
+    │  (a bug report or a   │
+    │   feature request)    │
+    │                       │
+    └───────────────────────┘
+               ⇩
+    ┌───────────────────────┐
+    │                       │
+    │  Open a Pull Request  │
+    │   (only after issue   │
+    │     is approved)      │
+    │                       │
+    └───────────────────────┘
+               ⇩
+    ┌───────────────────────┐
+    │                       │
+    │   Your changes will   │
+    │     be merged and     │
+    │ published on the next │
+    │        release        │
+    │                       │
+    └───────────────────────┘
+
+
+
+
+
+
+## How to create an issue
+-Navigate to the repository: 
+Go to the GitHub website and locate the repository where you want to create the issue. You can search for the repository or use the repository's URL to access it directly.
+
+-Access the "Issues" tab:
+ Once you're on the repository page, find the "Issues" tab in the top navigation bar and click on it. This tab will lead you to the issues management page.
+
+-Click on "New issue":
+ On the "Issues" page, you'll find a green button labeled "New issue" on the right-hand side. Click on it to proceed with creating a new issue.
+
+
+## How to create a pull request
+Creating a pull request on GitHub involves the following steps:
+
+1.Fork the repository.
+
+2.Create a new branch for your changes.
+
+3.Make the necessary modifications and test them.
+
+4.Commit and push your changes to your forked repository.
+
+5.Access the original repository and click on "New pull request".
+
+6.Select the appropriate branches for the base and head.
+
+7.Review and provide details about your changes.
+8.Create the pull request.
```

Comment:
Check the preview of this line

Reply:
sir can you explain in detail what you meant here?